spring-integration-jdbc/src/main/resources/org/springframework/integration/jdbc/schema-h2-json.sql

```diff
@@ -0,0 +1,61 @@
+CREATE TABLE INT_MESSAGE  (
+    MESSAGE_ID    CHAR(36)     NOT NULL,
+    REGION        VARCHAR(100) NOT NULL,
+    CREATED_DATE  TIMESTAMP    NOT NULL,
+    MESSAGE_BYTES CLOB,
```

Comment:
I think we should drop all of these extra schema scripts and just document how to change the target RDBMS to support JSON serialization instead.